lib/hanami/cli/generators/app/action_context.rb

```diff
@@ -6,50 +6,72 @@
 module Hanami
   module CLI
     module Generators
+      # @since 2.0.0
+      # @api private
       module App
+        # @since 2.0.0
+        # @api private
```

Comment:
@jodosha I marked this as `@api private` (with a view to having only the minimum required public API for this gem) but later noticed that these are also used by hanami-rspec.

Do you think they should be public API? Or are we happy just keeping these as internal for now?

Reply:
@timriley Mark as private.
Even if it's `hanami-rspec`, it's private for us as Hanami team.
I don't expect other people to use this class.